heroku/cli
# refactor: migrate spaces:vpn:wait to oclif/core

[Work Item](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001b1MLGYA2/view)

## Notes
- I added logic for printing the result as json since there is a flag for it
- I removed the `name` arg, favoring the `name` flag instead. The `name` arg was hidden and undocumented.
- I made the `space` and `name` flags required, and therefore was able to remove the logic that checked for them.

## Testing
- Create a private space
- Add a vpn connection (see the PR for `spaces:vpn:connect` for instructions)
- While the connection is provisioning, in a separate terminal window, run `spaces:vpn:wait -s SPACE_NAME -n VPN_CONNECTION_NAME`
    - If the connection provisions quickly, you may just get a message that says "VPN has been allocated". This is OK, it still means the command is working.
